src/components/Table.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+
+class Table extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <table className="table">
+        <tr>
+          <td>AC</td>
+          <td>+/-</td>
+          <td>%</td>
+          <td>+</td>
+        </tr>
+        <tr>
+          <td>7</td>
+          <td>8/-</td>
+          <td>9</td>
+          <td>*</td>
+        </tr>
+        <tr>
+          <td>4</td>
+          <td>5/-</td>
+          <td>6</td>
+          <td>-</td>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2/-</td>
+          <td>3</td>
+          <td>/</td>
+        </tr>
```

Comment:
- [ ] Kindly make sure you remove the some typing errors in some of your buttons to match the expected design.
![Screenshot from 2022-09-07 16-32-40](https://user-images.githubusercontent.com/83042742/188906267-0bb51260-6a67-400a-a394-dc6a4a1ddad1.png)